### ISSUE TITLE
vedirect-interface: add support for 24 bit version numbers

### ISSUE THREAD

**Issue by mpvader** [victronenergy/venus#212]
See [vregs.xml in velib](https://github.com/victronenergy/velib/blob/master/data/protocols/vregs.xml#L2126) for the new definition.

```
# dbus -y com.victronenergy.battery.ttyO2 /FirmwareVersion GetText
value = 'v3.08'
```

**Comment by mpvader** [comment]
Thiemo warned about some tricky bits:
1. Are we sure that every device gives a correct answer for VE_REG_APP_VERSION?
2. What do we do with the BMV60x where we cannot send hex? It is not really a problem because there is no 3 byte version inside the BMV, but we do need conversion from the 2 byte text frame version.
3. Do we need to ignore the version in the text frame for devices that cannot talk hex? I am not sure if the ve.direct stack in vedirect_interface has knowledge if hex trafic is allowed.

**Comment by izak** [comment]
A new text item `FWE` was added that contains the 24-bit version number. This answers all the tricky questions above.

Implementation [here][1] and [here][2].

[1]: https://git.victronenergy.com/victron/vedirect_consumer/tree/un24-versions
[2]: https://github.com/victronenergy/vedirect_interface/commits/un24-versions

**Comment by mpvader** [comment]
Izak is done and is waiting for review or inclusion

**Comment by izak** [comment]
Included in Venus v2.30~24.
